templates/puppetboard-ingress.yaml

```diff
@@ -1,8 +1,10 @@
-{{- if .Values.puppetboard.enabled }}
-{{- if .Values.puppetboard.ingress.enabled }}
+{{- if and .Values.puppetboard.enabled .Values.puppetboard.ingress.enabled }}
 {{- $releaseName := .Release.Name -}}
 {{- $serviceName := "puppetdb" }}
 {{- $servicePort := .Values.puppetboard.port -}}
+{{- $pathType := .Values.puppetboard.ingress.pathType | default "ImplementationSpecific" -}}
+{{- $apiIsStable := eq (include "puppetserver.ingress.isStable" .) "true" -}}
+{{- $ingressSupportsPathType := eq (include "puppetserver.ingress.supportsPathType" .) "true" -}}
 apiVersion: networking.k8s.io/v1
```

Comment:
Shouldn't this use `puppetserver.ingress.apiVersion`?

Reply:
Yup - fixed. Thanks.